gnovm/pkg/gnolang/preprocess.go

```diff
@@ -2892,10 +2898,19 @@ func doConvertType(store Store, last BlockNode, x *Expr, t Type) {
 //	convert to that even if right is a named type.
 //	case 2: isNamedConversion is called within evaluating make() or new()
 //	(uverse functions). It returns TypType (generic) which does have IsNamed appropriate
+//
+// This function also checks for the use of blank identifier "_" as a value or type,
+// which is not allowed. If both xt and t are nil, it panics with an appropriate error message.
 func isNamedConversion(xt, t Type) bool {
+	if xt == nil && t == nil {
+		panic("cannot use _ as value or type")
```

Comment:
did not see any test targeting this.

Reply:
https://github.com/gnolang/gno/pull/2699/commits/d8d53a11c81dab6e6e259df9b0d5489eeabec90c

I wrote new test for `isNamedConversion` function to replace the tests that were previously run through the machine. I checked that this cover everything.

Reply:
can you provide a file test like this:
```go
package main

type (
	nat  []word
	word int
)

func main() {
	var a nat
	b := []word{0}
	a = b

	println(a)
}
```

to better illustrate why are these logic(added) needed?

Reply:
what I mean here is that I did not find a situation that xt and t are both nil in the context of `isNamedConversion`.

Reply:
It was definitely an unnecessary check. I remember adding it as defensive logic just in case, but I will delete this. 👍